imports/ui/pages/student_results.jsx

```diff
@@ -7,8 +7,6 @@ import React, { Component } from 'react'
 // import ReactDOM from 'react-dom'
 import { createContainer } from 'meteor/react-meteor-data'
 import { _ } from 'underscore'
-import $ from 'jquery'
-import dl from 'datalib'
```

Comment:
We should look at where datalib is being used and remove it if reasonable. I found a couple of places where it was being used wrong and incorrectly calculating statistics, so I just wrote some likely less-efficient, but correct, code.  I found that where it was being used, it was too complicated to understand what it was actually doing; especially for calculations, I prefer legibility of the code over conciseness....

Reply:
That seems to be true. I cannot find any other references. I'll remove from the package.json as well.